### ISSUE TITLE
Handle images better

### ISSUE THREAD

**Issue by brittneypostma** [Svelte-Sirens/svelte-sirens#86]
Optimize our images with something like [Vite Imagetools](https://www.npmjs.com/package/vite-imagetools)

**Comment by spences10** [comment]
Did you see that Vite turns assets into a hashed URL?

Here's Rich talking about it at ViteConf:

https://youtu.be/-OMPfr56kXI?t=400

And the reference in the example project: 

https://github.com/sveltejs/kit-template-default/blob/455c5800c82cd4bc7ab266af13012b3bb72f89e9/src/routes/%2Bpage.svelte#L4

**Comment by brittneypostma** [comment]
Yes I did, I'm going through the code base right now with a move to Skelton UI and am moving images I see over to use that.
